notebooks/time-to-merge-prediction/workshop/deployment-config.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: machinelearning.seldon.io/v1
+kind: SeldonDeployment
+metadata:
+  name: your-username
+spec:
+  name: your-username
+  predictors:
+  - componentSpecs:
+    - spec:
+        containers:
+        - image: quay_image_repo
```

Comment:
we do not expect users to s2i build their image. We should link a working image here which they should not need to modify